package.json

```diff
@@ -21,13 +21,13 @@
     "url": "https://github.com/guillaumepotier/validator.js"
   },
   "devDependencies": {
-    "mocha": "*",
     "expect.js": "*",
     "grunt": "~0.4.2",
-    "grunt-contrib-uglify": "~1.0.1",
     "grunt-contrib-clean": "~0.5.0",
+    "grunt-contrib-uglify": "~1.0.1",
+    "grunt-npm2bower-sync": "~0.3.0",
```

Comment:
This reorder was done automatically when running `yarn add`. IMO we might as well leave it sorted, to avoid such unrelated changes in the future when performing similar tasks.